ui/widgets/autocomplete.js

```diff
@@ -233,17 +243,21 @@ $.widget( "ui.autocomplete", {
 				// so we set a flag to know when we should ignore the blur event
 				this.cancelBlur = true;
 				this._delay( function() {
-					delete this.cancelBlur;
 
-					// Support: IE 8 only
-					// Right clicking a menu item or selecting text from the menu items will
-					// result in focus moving out of the input. However, we've already received
-					// and ignored the blur event because of the cancelBlur flag set above. So
-					// we restore focus to ensure that the menu closes properly based on the user's
-					// next actions.
+					// Support: IE
```

Comment:
We don't use unversioned support comments like this. Presumably you're hitting a different scenario (not right clicking), so the following comment explaining the support comment isn't right anymore.